### ISSUE TITLE
custom template as page item

### ISSUE THREAD

**Issue by Taha19861986** [Pretius/apex-nested-reports#26]
I'am using a page item as a source to the custom template attribute because it reaches the maximum character limit. in page load i'am bind this page item with my template html. but it requires me to submit the page again for the plugin to detect the page item value. is there any way to refresh the plugin custom template attribute with the page item value without submitting the page again.

**Comment by bostrowski** [comment]
@Taha19861986 

please reproduce your use case scenario at apex.oracle.com. I would like to answer like:

> just set the value of the item before page is loaded

but lets see what you have already done.

Recreate the use case scenario at apex.oracle.com to the point you have stuck with and send me login (developer) credentials to ostrowski.bartosz@gmail.com

**Comment by StormingMovies** [comment]
Taha, are you using APEX? if so, you can submit just the content of your item without submiting the whole page. Just create a previous pl/sql action with a "begin null; end;" and the item's value gets submitted and can be used in the template

**Comment by bostrowski** [comment]
@StormingMovies 
I think you are missing the point. @Taha19861986 has described the case where he/she had to override the 4000 characters limit for plugin attribute and because of that he/she used substitution string which is evaluated only once when the page loads. 

Your suggestions doesn't applu because APEX evaluates substitutions strings when page is rendered. It means that the page must be reloaded so the plug-in can use the new value of the item that was referenced through substitution string (&PX_ITEM_NAME.).

Still, without @Taha19861986 reproducing the issue at apex.oracle.com I can't say more. Its been 3 months from his/her issue and I think I will close this thread as it is abandoned by the author.
